src/main/resources/fileTemplates/testMeTests/JUnit4 & Mockito.java.ft

```diff
@@ -29,7 +29,7 @@ public class ${CLASS_NAME} {
 #if($TestSubjectUtils.shouldBeTested($method))
 
     @Test
-    public void #renderTestMethodName($method.name)() throws Exception {
+    public void #renderTestMethodName($method.name)() #if($method.hasException()) throws $method.methodExceptionTypes #end{
```

Comment:
remove redundant spaces. before `#if` and `#end`. comment relevant for other templates as well